server/serverless.yml

```diff
@@ -108,6 +108,7 @@ provider:
     AUTOMATIC_NOTIFICATIONS_TOPIC_TITLE: ${self:provider.stage}-cloud-crm-automatic-notifications
     AUTOMATIC_NOTIFICATIONS_TOPIC_ARN:
       Ref: ContactManagementTopic
+    BASE_URL: perfsys-cloud-crm-ui-${self:provider.stage}.s3-website-eu-west-1.amazonaws.com
```

Comment:
@evasilkovich please rename BASE_URL to WEB_UI_BASE_URL. As this is a link to UI that. A

Reply:
@evasilkovich Also can you replace eu-west-1 with a sls variable ?